include/mesh.h

```diff
@@ -202,6 +209,8 @@ class Mesh {
   Map<NodeBase<Tdim>> map_nodes_;
   //! Container of cells
   Container<Cell<Tdim>> cells_;
+  //! Map of cells for fast retrieval
```

Comment:
Don't do this, instead iterate to find the cell pointer and then assign the velocity constraint.